recipes/csaps/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is in the tarball, no need to add it here.

Reply:
I removed the file.